smdebug/tensorflow/collection.py

```diff
@@ -133,7 +138,10 @@ def __init__(self, collections=None, create_default=True):
         if create_default:
             for n in DEFAULT_TF_COLLECTIONS:
                 self.create_collection(n)
-            self.get(CollectionKeys.BIASES).include("bias")
+            self.get(CollectionKeys.BIASES).include("^(?!gradient).*bias")
+            self.get(CollectionKeys.WEIGHTS).include("^weights/.*/((?!bias).)*$")
```

Comment:
My regex-fu is limited, but looks like this will exclude bias gradients in CollectionKeys.BIASES, but include weight gradients in CollectionKeys.WEIGHTS. Is that what we want?

Reply:
weight gradients are called "gradients/dense/kernelGrad".
bias gradients are called "gradients/dense/biasGrad". hence, only bias needs a special regex. similar to PT